packages/parser-html/src/parser.ts

```diff
@@ -26,82 +26,30 @@ try {
     // `canvas` is not installed, nothing to do
 }
 
-import { JSDOM } from 'jsdom';
-import { JSDOMAsyncHTMLElement, JSDOMAsyncWindow } from 'hint/dist/src/lib/types/jsdom-async-html';
-import { Event, FetchEnd, Parser, TraverseDown, TraverseUp } from 'hint/dist/src/lib/types';
+import { Parser, FetchEnd } from 'hint/dist/src/lib/types';
 import { Engine } from 'hint';
+import createAsyncWindow from 'hint/dist/src/lib/utils/dom/create-async-window';
```

Comment:
Should this now be `createHTMLDocument` instead? I was under the impression `parser-html` wouldn't be referencing `jsdom` with these changes...

Reply:
No, in this case. The `parser-html` needs the jsdom so we can have an `evalute`.